src/main/java/org/team1540/robot2024/subsystems/vision/AprilTagVisionIOLimelight.java

```diff
@@ -0,0 +1,57 @@
+package org.team1540.robot2024.subsystems.vision;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Pose3d;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.wpilibj.Timer;
+import org.team1540.robot2024.util.vision.LimelightHelpers;
+
+public class AprilTagVisionIOLimelight implements AprilTagVisionIO {
+    private final String name;
+
+    public AprilTagVisionIOLimelight(String name, Pose3d cameraOffsetMeters) {
+        this.name = name;
+        LimelightHelpers.setCameraPose_RobotSpace(
+                name,
+                cameraOffsetMeters.getX(),
```

Comment:
What happens when these change as our elevator moves up

Reply:
Isn't the limelight going to be mounted on a stationary position? If not I could add a method to set the pose.

Reply:
Funny joke Alvin. Add the method

Reply:
:skull: ok fine method added